src/data_loaders.py

```diff
@@ -49,8 +48,10 @@ def data_gen_standard(id, audio_repr_path, gt, pack):
         # let's deliver some data!
         if sampling == 'random':
             for i in range(0, param_sampling):
-                random_frame_offset = random.randint(
-                    0, frames_num - config['xInput'])
+                random_uniform = np.random.random()
```

Comment:
Why not use [numpy.random.randint](https://docs.scipy.org/doc/numpy-1.16.0/reference/generated/numpy.random.randint.html)?
(I think, to match the behaviour of `random.randint` you need to add `+1` to the upper bound)

Reply:
There was a reason for this, but it was not evident. I added an explanatory comment.